WooCommerce/Classes/ViewRelated/Orders/Order Creation/PaymentSection/OrderPaymentSection.swift

```diff
@@ -82,35 +88,36 @@ struct OrderPaymentSection: View {
             }
 
             addCouponRow
+                .zIndex(1)
```

Comment:
This is an interesting one: My understanding is that you'd need a ZStack in order to apply `.zIndex`, but that [doesn't seem to be the case](https://developer.apple.com/documentation/swiftui/view/zindex(_:)). For some reason I wasn't able to make the tooltip appear properly on top of the other views without transparency (despite setting opacity specifically, or trying different alternatives) and using this property is the only thing that fixed it.

Reply:
Added internal comment: 5122b2c900be724137b3799a4205bdf57f091678